config/leviton/DZPA1

```diff
@@ -0,0 +1,72 @@
+• Enter Programming Mode by holding the control button for 7 seconds, the Locator LED will blink amber.
+• The Decora SmartTM Z-Wave® device is ready to add to the Z-Wave® network.
+• Follow directions in the Z-Wave® controller to enter learn mode.
+• Tap the control button one time. The Locator LED will quickly flash green.
+The Z-Wave® controller will begin to pair with the Decora Smart device.
+• Upon successful addition to the network the LED will turn off and then blink green 3 times.
+• If the adding process is not successful the LED will flash red 3 times.</MetaDataItem>
+    <MetaDataItem name="ProductManual">https://products.z-wavealliance.org/ProductManual/File?folder=&filename=MarketCertificationFiles/1959/DI-000-DZPA1-02B-X4.pdf</MetaDataItem>
+    <MetaDataItem name="ResetDescription">When removing a device from a network it is best practice to use the Exclusion process. In situations where a device needs to be returned to factory default follow the following steps. A reset should only be used when a controller is inoperable or missing:
+• Holding the control button for a total of 14 seconds.
+- After the first 7 seconds the LED turns amber.
+- The Locator LED will quickly flash red/amber after 14 seconds.
+• Release the control button and the device will reset.</MetaDataItem>
+    <MetaDataItem name="Name">Plug-In Outlet</MetaDataItem>
+    <MetaDataItem id="0001" name="FrequencyName" type="3401">U.S. / Canada / Mexico</MetaDataItem>
+    <ChangeLog>
+      <Entry author="Jim Kaiser" date="22 June 2020" revision="1">Fixed incomplete clone of DZ15S to DZPA1 - https://products.z-wavealliance.org/products/1959/xml</Entry>
+    </ChangeLog>
+    <MetaDataItem id="0001" name="ZWProductPage" type="3601">https://products.z-wavealliance.org/products/1959/</MetaDataItem>
+    <MetaDataItem id="0001" name="Identifier" type="3601">DZPA1</MetaDataItem>
+    <MetaDataItem id="0001" name="FrequencyName" type="3601">U.S. / Canada / Mexico</MetaDataItem>
+  </MetaData>
+  <!-- Configuration Parameters -->
+  <CommandClass id="112">
+    <Value genre="config" index="7" label="Locator LED Status" max="255" min="0" size="1" type="list" units="" value="255">
+      <Help>Change the mode of the LED to Status Mode where the LED is illuminated when the load is On, Locator Mode where the LED is illuminated when the Load is Off, or turns off the LED completely for any other setting values.</Help>
+      <Item label="Off" value="0"/>
+      <Item label="Status Mode" value="254"/>
+      <Item label="Locator Mode" value="255"/>
+    </Value>
+  </CommandClass>
+  <!-- Association Groups -->
+  <CommandClass id="133">
+    <Associations num_groups="1">
+      <Group index="1" label="Lifeline" max_associations="5"/>
+    </Associations>
+  </CommandClass>
+  <CommandClass id="134">
+    <Compatibility>
+      <ClassGetVersionSupported>false</ClassGetVersionSupported>
```

Comment:
Why are we not getting a version? this indicates a "broken" device so just want to confirm

Reply:
This is a new device, finishing someone's copying of the DZ15S.xml to a new DZPA1.xml. Or, am I misunderstanding?